test/run_test.py

```diff
@@ -1808,6 +1809,17 @@ def __str__(self):
                 print_to_stderr("Emiting td_test_failure_stats")
                 emit_metric("td_test_failure_stats", test_stats)
 
+                # Is this an excessive number of things to emit? probably.  But
+                # it was also really easy to write
+                emit_metric(
+                    "td_number_ranking_experiment",
+                    {
+                        **get_experimental_ratings(selected_tests),
```

Comment:
It might be easier to write more future proof queries if you put this inside it's own column
```suggestion
                        "confidence_scores": **get_experimental_ratings(selected_tests),
```